README.md

```diff
@@ -0,0 +1,23 @@
+Reflection Activity 
+===============
+
+Reflection is a game designed around reflective symmetry. The user modifies an image until it has reflective symmetry around either the horizontal or vertical axis.
+
+How to use?
+===============
+Reflection is not a part of Sugar Desktop but can be added. Refer to the following links-
+
+* [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/), 
+* [How to use Sugar](https://help.sugarlabs.org/), and;
+* [How to use Reflection](https://wiki.sugarlabs.org/go/Activities/Reflection) 
```

Comment:
As part of maintenance, we also move Wiki documentation into the Help activity.

* https://help.sugarlabs.org/en/
* https://github.com/godiard/help-activity

Reply:
Defer via https://github.com/sugarlabs/reflection/issues/7